featurebyte/routes/common/base_table.py

```diff
@@ -174,6 +180,24 @@ async def _add_table_description_from_warehouse(
             )
         return document
 
+    async def _start_table_validation_task(self, table_document: TableDocumentT) -> Task:
+        await self.service.update_document(
+            document_id=table_document.id,
+            data=self.document_update_schema_class(  # type: ignore
+                validation=TableValidation(status=TableValidationStatus.PENDING)
+            ),
+            return_document=True,
```

Comment:
nit: we could remove `return_document=True` if we are not using the updated document.